bugsnag-android-core/src/androidTest/java/com/bugsnag/android/ClientTest.java

```diff
@@ -204,10 +207,20 @@ public void testClientUser() {
         assertNotNull(client.getUser().getId());
     }
 
+    @Test
+    public void testClientBreadcrumbRetrieval() {
+        client = generateClient();
+        client.leaveBreadcrumb("Hello World");
+        List<Breadcrumb> breadcrumbs = client.getBreadcrumbs();
+        List<Breadcrumb> store = new ArrayList<>(client.breadcrumbState.getStore());
+        assertEquals(store, breadcrumbs);
+        assertNotSame(store, breadcrumbs);
+    }
+
```

Comment:
A further check that adding > maxBreadcrumbs returns a list of the expected size and order would help ensure the javadoc is correct.

Reply:
This requirement is already covered in `ClientTest#testMaxBreadcrumbs`

Reply:
Not through the public interface - perhaps the test should be updated to use the new method rather than `client.breadcrumbState.getStore()`?